### PR TITLE
Removed hardcoded sizes of components of Manage Tags dialog

### LINKED ISSUE #6512: Buttons of Manage Tags dialog are too small
**Describe the bug**
The "Cancel" and "Delete" button of the Manage Tags dialog are too small for their text to be fully visible.

**To Reproduce**
1. Go to any entry in the History Tab.
2. Right click, go to "Manage Tags..."
3. The buttons' text is not fully visible.

**Screenshots**

Screenshot from 2021-04-04 17-17-16

**Expected behavior**
The buttons should be large enough for their text to display properly.

**Software versions**
 - ZAP: 2.10.0
 - OS: Ubuntu 20.04
 - Java: OpenJDK 11

### PR DESCRIPTION
**Pull request by pranavsaxena17**
The hardcoded sizes of the Manage Tags dialog window and buttons present in it have been removed.

Fixes #6512

### CODE REVIEW

**Comment by thc202** [comment]
The correct fix is to remove the hardcoded sizes.

**Comment by pranavsaxena17** [comment]
@thc202 Would that include removing all three values for the buttons (that would lead to different sized buttons) or should I keep the minimum size values?

**Comment by thc202** [comment]
Yes, all sizes should be removed, it should be used the layout to do that instead.

**Comment by pranavsaxena17** [comment]
@thc202 Could you kindly hop on IRC for a minute?

File: zap/src/main/java/org/zaproxy/zap/extension/history/ManageTagsDialog.java [hunk 1]
```diff
@@ -42,8 +42,8 @@
 public class ManageTagsDialog extends AbstractDialog {
 
     private static final long serialVersionUID = 1L;
-    private static final int panelWidth = 250;
-    private static final int panelHeight = 300;
+    private static final int panelWidth = 500;
+    private static final int panelHeight = 350;
```
Comment (thc202) [inline zap/src/main/java/org/zaproxy/zap/extension/history/ManageTagsDialog.java:46]: These can be removed as well.

**Comment by thc202** [comment]
The PR/commit should be updated to reflect the latest changes.

File: zap/src/main/java/org/zaproxy/zap/extension/history/ManageTagsDialog.java (hunk 1 above)
Comment (pranavsaxena17) [reply zap/src/main/java/org/zaproxy/zap/extension/history/ManageTagsDialog.java]: Done

**Comment by pranavsaxena17** [comment]
@thc202 Done

**Comment by kingthorin** [comment]
The [initial PR comment](https://github.com/zaproxy/zaproxy/pull/6513#issue-608532230) should be edited to align with  the current state of the change/PR.

**Comment by pranavsaxena17** [comment]
@kingthorin Tweaked

**Review by kingthorin — APPROVED** [review]
Seems fine on Win10, Java 11. Flaflaf Dark Theme.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**